File: app/src/main/java/ch/unstable/ost/utils/LogUtils.java
package ch.unstable.ost.utils;


import com.google.gson.Gson;
import com.google.gson.GsonBuilder;
import com.google.gson.JsonElement;

public enum LogUtils {
    ;

    /**
     * Get a pretty json from an json element.
     * @param element the json element
     * @return the pretty json
     */
    public static String prettyJson(JsonElement element) {
        Gson gson = new GsonBuilder()
                .setPrettyPrinting()
                .create();

        return gson.toJson(element);
    }
}

File: app/src/main/java/ch/unstable/ost/utils/LogUtils.kt
package ch.unstable.ost.utils

import com.google.gson.GsonBuilder
import com.google.gson.JsonElement

object LogUtils {
    /**
     * Get a pretty json from an json element.
     * @param element the json element
     * @return the pretty json
     */
    fun prettyJson(element: JsonElement?): String {
        val gson = GsonBuilder()
                .setPrettyPrinting()
                .create()
        return gson.toJson(element)
    }
}